### PR TITLE
Rename import paths: "github.com/gogits/gogs" -> "github.com/go-gitea/gitea"

### PR DESCRIPTION
**Pull request by andreynering**

### CODE REVIEW

**Review by tboerger — APPROVED** [review]
I have not checked if everything have been replaced but looks good to me

File: cmd/serve.go [hunk 1]
```diff
@@ -18,11 +18,11 @@ import (
 	gouuid "github.com/satori/go.uuid"
 	"github.com/urfave/cli"
 
-	"github.com/gogits/gogs/models"
-	"github.com/gogits/gogs/modules/base"
-	"github.com/gogits/gogs/modules/httplib"
-	"github.com/gogits/gogs/modules/log"
-	"github.com/gogits/gogs/modules/setting"
+	"github.com/go-gitea/gitea/models"
```
Comment (strk) [inline cmd/serve.go:21]: are relative imports not supported ?

**Review by strk — COMMENTED** [review]
If there's a way to use relative imports, it would be simpler to maintain in case of another switch

**Review by bkcsoft — APPROVED** [review]
LGTM

File: cmd/serve.go (hunk 1 above)
Comment (andreynering) [reply cmd/serve.go:21]: It is supported but not recommended

**Comment by jhasse** [comment]
You'll need search-and-replace anyway ;)

**Comment by metalmatze** [comment]
LGTM, small rant: Golang team now adds aliases, but we still don't have a tool for this. :)

**Comment by tboerger** [comment]
LGTM

**Comment by bkcsoft** [comment]
LGTM

(need to do it again -.-)

**Comment by bkcsoft** [comment]
LGTM

Third times the charm!

**Pull request merged**